docs/recipes/custom-gatherer-puppeteer/test.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")"
```

Comment:
```suggestion
# Make sure we're in this `docs/recipes/customer-gatherer-puppeteer` directory
cd "$(dirname "$0")"
```

Reply:
wait nevermind I just looked up the pattern for this for the other PR and now I'm not sure what this is supposed to be doing haha :)

Reply:
your suggested comment is correct tho